main_test.go

```diff
@@ -59,7 +66,7 @@ func Test_run(t *testing.T) {
 	}
 }
 
-var re = regexp.MustCompile(`generated by .*deep-copy.*; DO NOT EDIT.`)
+var re = regexp.MustCompile(`generated by .*deep[-_]copy.*; DO NOT EDIT.`)
```

Comment:
Out of curiosity, why was the change on this line required?

Reply:
Sorry for the delay in responding.

The test was failing without it at the time I applied this patch, but when I run it again now, it passes for some reason.
So I revert it at 54f8df4